packages/workspace/src/workspace/hidden_values.ts

```diff
@@ -188,10 +189,25 @@ export const mergeWithHidden = async (
   return mergeElements(awu(hiddenStateElements).concat(workspaceElementsWithHiddenParts))
 }
 
+// Check if the given path is nested in a list
+const isPathInList = (path: ElemID | undefined): boolean => some(path?.getFullNameParts(), isIndexPathPart)
```

Comment:
prefer builtin methods over lodash when possible

```suggestion
const isPathInList = (path: ElemID | undefined): boolean => path?.getFullNameParts().some(isIndexPathPart) ?? false
```

Reply:
Done